tests/outputs/a_manurev/refereeReports.yaml

```diff
@@ -1,21 +1,27 @@
 1902cl_actaArev:
   claimed_found_what:
+    -
```

Comment:
normally we would write it as 
```
claimed_found_what: []
```
though it is just an esthetic choice (more readable maybe?)